<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Profile</class>
 <widget class="QDialog" name="Profile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>364</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>341</width>
     <height>321</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout" columnstretch="0,0">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="horizontalSpacing">
     <number>6</number>
    </property>
    <item row="4" column="0">
     <widget class="QLabel" name="accessLevel_label">
      <property name="text">
       <string>УД к Архиву:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Никнейм:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QPushButton" name="saveChange_pushButton">
      <property name="text">
       <string>Сохранить изменения</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lin_lineEdit"/>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="nickname_lineEdit"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>ЛИН:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="date_lineEdit"/>
    </item>
    <item row="8" column="1">
     <widget class="QPushButton" name="exitProfile_pushButton">
      <property name="text">
       <string>Выйти из профиля</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QPushButton" name="deleteAccount_pushButton">
      <property name="text">
       <string>Удалить профиль</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QPushButton" name="changePassword_pushButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Сменить пароль</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Статус:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Дата регистрации:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="status_comboBox">
      <item>
       <property name="text">
        <string>Владелец</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Директор</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Сотрудник</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Стажёр</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QComboBox" name="accessLevel_comboBox">
      <item>
       <property name="text">
        <string>Владелец</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Администратор</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Модератор</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Пользователь</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="5" column="1">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
